Api/Startup.cs

```diff
@@ -60,14 +76,18 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
+            app.UseProblemDetails();
```

Comment:
I remember reading somewhere that this problem details package lets you set up so that you can still get developer errors in dev, think it was in one of the links I sent, would be a good idea to set that up

Reply:
Do you mean developer errors in staging? I assume you don't mean production, since we wouldn't want all this information to be sent out on the production branch right?